### ISSUE TITLE
README is lacking instructions for installation

### ISSUE THREAD

**Issue by wfendler** [cratejoy/cratejoy-themes#1]
There doesn't seem to be any information available (both here, and on your [documentation](http://cratejoy.com/documentation/) on your website) on how to install a new theme.

**Comment by makesitgood** [comment]
Hi William, you're right.  There's not a publicly available method for doing so right now.  We'll be exposing that functionality, but currently  if you have something custom you'd like installed you can contact support@cratejoy.com and we'll help you out.

The most common use case right now is to install an existing theme from the design store as a starting point, then customize from that.

**Comment by wfendler** [comment]
Okay great, thanks! I'll reach out at that email address.

**Comment by kfrz** [comment]
I'm still struggling with this issue. 

**What I Did:**
I have exported a theme from the Cratejoy platform, which I've modified locally,

**What Should Happen:**
I can upload the theme and import into Cratejoy.

**What Actually Happens:**
I can't upload themes via import function. Using FTP is an incomplete solution as other bugs prevent full uploading

@expl0ratory any updates on this as per your comment above?
